src/qt/clientmodel.h

```diff
@@ -10,7 +10,7 @@
 
 #include <atomic>
 #include <memory>
-#include <sync.h>     // for Mutex
+#include <sync.h>
```

Comment:
In commit "Added best block hash to the NotifyHeaderTip and NotifyBlockTip signals." (20049ec7eac1f555dde04026594c4da4b1b80c8b)

Would recommend squashing the two commits to a single one

- 6002d1185db89af124a257c894195a7f1b04c1f2 Added best block hash to the NotifyHeaderTip and NotifyBlockTip signals. (1/5)
- 20049ec7eac1f555dde04026594c4da4b1b80c8b Added best block hash to the NotifyHeaderTip and NotifyBlockTip signals. (2/5)

Reply:
yeah absolutely. Same as the other point, rebase issue that i totally missed. Squashing it.